controlplane/eks/controllers/awsmanagedcontrolplane_controller.go

```diff
@@ -238,11 +248,21 @@ func (r *AWSManagedControlPlaneReconciler) reconcileNormal(ctx context.Context,
 	return reconcile.Result{}, nil
 }
 
-func (r *AWSManagedControlPlaneReconciler) reconcileDelete(_ context.Context, managedScope *scope.ManagedControlPlaneScope) (_ ctrl.Result, reterr error) {
+func (r *AWSManagedControlPlaneReconciler) reconcileDelete(ctx context.Context, managedScope *scope.ManagedControlPlaneScope) (_ ctrl.Result, reterr error) {
 	managedScope.Info("Reconciling AWSManagedClusterPlane delete")
 
 	controlPlane := managedScope.ControlPlane
 
+	numDependencies, err := r.dependencyCount(ctx, managedScope)
+	if err != nil {
+		return reconcile.Result{}, fmt.Errorf("error getting controlplane dependencies %s/%s: %w", controlPlane.Namespace, controlPlane.Name, err)
```

Comment:
```suggestion
                r.Log.Error(err, "error getting controlplane dependencies")
		return reconcile.Result{}, err)
```
Usual thing about getting the line of code the error comes from before it returns to controller runtime.

Reply:
Thanks @randomvariable, changed